app/connector/mongodb/src/test/java/io/syndesis/connector/mongo/MongoDBConnectorInsertTest.java

```diff
@@ -47,11 +49,16 @@ public void mongoInsertTest() {
         // When
         String uniqueId = UUID.randomUUID().toString();
         String message = String.format("{\"test\":\"unit\",\"uniqueId\":\"%s\"}", uniqueId);
+        Document doc = Document.parse(message);
         // Given
-        template().sendBody("direct:start", message);
+        Document result = Document.parse(template.requestBody("direct:start", message, String.class));
```

Comment:
Might be easier to debug if the result of `template.requestBody(...)` was put in a variable.